modules/bitgo/test/v2/unit/internal/tssUtils/common.ts

```diff
@@ -34,9 +34,9 @@ export async function nockCreateTxRequest(params: { walletId: string, requestBod
     .reply(200, params.response);
 }
 
-export async function nockGetTxRequest(params: {walletId: string, txRequestId: string, response: unknown}): Promise<nock.Scope> {
+export async function nockGetTxRequest(params: {walletId: string, txRequestId: string, response: unknown, notPersist?: boolean}): Promise<nock.Scope> {
   return nock('https://bitgo.fakeurl')
-    .persist(true)
+    .persist(params.notPersist)
```

Comment:
This changes correct behaviour in the mocks which is reused for eddsa as well, but should be fine if all tests passes